webapp/src/views/EditPage.vue

```diff
@@ -200,6 +205,12 @@ export default {
         this.lastModified = formatDistanceToNow(save_date, { addSuffix: true });
       }
     },
+    async getBlocksInfo() {
+      if (!this.blocksInfos) {
+        let data = await getBlocksInfos();
+        this.$store.commit("setBlocksInfos", data);
```

Comment:
I think we should do the store commit inside `server_fetch_utils` so that it can be reused elsewhere without needing the edit page loaded directly